test/TestParser.hs

```diff
@@ -0,0 +1,112 @@
+module TestParser (tests) where
+
+import Data.Either (isLeft)
+import Data.Text as T
+import Data.Text.Arbitrary ()
+import Lhx.Parser qualified as LP
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.HUnit (testCase, (@?), (@?=))
+import Test.Tasty.QuickCheck as QC
+
+type Title = String
+
+type MessageIfFail = String
+
+type ParsedString = T.Text
+
+type ExpectedValue = [LP.Chunk]
```

Comment:
This is a bad alias! In the library we already have one: `Template`. Why do we need another one just in tests? And why "Value"? All values we need to test are templates? I don't think so.